test/e2e/app-dir/app-static/app/ssg-draft-mode/[[...route]]/page.js

```diff
@@ -1,11 +1,16 @@
 import { draftMode } from 'next/headers'
 
-export default function Page() {
+export default async function Page() {
+  const data = await fetch(
+    'https://next-data-api-endpoint.vercel.app/api/random'
```

Comment:
In the future, we should move this deployment to a shared team such as `curated-tests`